### PR TITLE
Compact chip

### PR DESCRIPTION
**Pull request by xwatkins**

### CODE REVIEW

File: src/components/chip.jsx [hunk 1]
```diff
@@ -3,8 +3,13 @@ import PropTypes from 'prop-types';
 import RemoveIcon from '../svg/times.svg';
 import '../styles/components/chip.scss';
 
-const Chip = ({ children, onRemove, className, disabled }) => (
-  <span className={`chip ${disabled ? 'chip--disabled' : ''} ${className}`}>
+const Chip = ({ children, onRemove, className, disabled, compact, title }) => (
+  <span
+    className={`chip ${disabled ? 'chip--disabled' : ''} ${
```
Comment (aurel-l) [inline src/components/chip.jsx:8]: Maybe we can start using the `classnames` library now that Mahdi added it somewhere else in the codebase. My OCD is getting triggered when I see the 3 white spaces in the generated class in the corresponding tests 😅

Comment (xwatkins) [reply src/components/chip.jsx:8]: The order of the world is now balanced 🧘‍♂️

Comment (aurel-l) [reply src/components/chip.jsx:8]: you didn't push yet, though?

Comment (aurel-l) [reply src/components/chip.jsx]: 🧘‍♂️

**Pull request merged**